tasks/libs/common/utils.py

```diff
@@ -532,7 +532,8 @@ def gitlab_section(section_name, collapsed=False, echo=False):
     """
     - echo: If True, will echo the gitlab section in bold in CLI mode instead of not showing anything
     """
-    section_id = section_name.replace(" ", "_").replace("/", "_")
+    # Specific characters can prevent the generation of the section
+    section_id = re.sub(r"[ :/\\]", "_", section_name)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What about using

```python
section_id = section_name.replace(" ", "_").replace("/", "_").replace(":", "_").replace("\\", "_")
```

? Or, would add in the comment what `re.sub` is doing, I find `replace` more explicit, but more verbose

Reply:
Yes I wanted to prevent 4 calls of `replace` when `sub` can do everything in one call. I can rephrase the comment with:
```
# Replace with re.sub every special character (" ", "/", "\" and ":") which prevent the section generation
```
wdyt?

Reply:
Perfect!